src/components/Thumbnail/Thumbnail.js

```diff
@@ -138,10 +152,15 @@ class Thumbnail extends React.PureComponent {
             width: THUMBNAIL_SIZE,
             height: THUMBNAIL_SIZE,
           }}
-          ref={this.thumbContainer}
-          onDragStart={this.onDragStart}
-          draggable={isDraggable}
-        />
+        >
+          <div
+            className="thumbnail"
+            ref={this.thumbContainer}
+            onClick={this.handleClick}
```

Comment:
Is it necessary to have a click handler here? We already have one at line 146.

Reply:
Yea this isn't needed anymore